README.yaml

```diff
@@ -69,16 +66,31 @@ usage: |-
   Here's how to invoke this example module in your projects
 
```

Comment:
We need to add a fully annotated configuration file example

Reply:
(in the readme, not just link to it)

Reply:
@osterman All done if you want to have a look